### PR TITLE
Cherrypick 6.11.z manifest helper

### PR DESCRIPTION
**Pull request by shweta83**
Manual Cherrypick of following PRs

- https://github.com/SatelliteQE/robottelo/issues/10409
- https://github.com/SatelliteQE/robottelo/issues/10705
- https://github.com/SatelliteQE/robottelo/issues/10706
- https://github.com/SatelliteQE/robottelo/issues/10847
- https://github.com/SatelliteQE/robottelo/issues/10849

### CODE REVIEW

File: pytest_fixtures/component/taxonomy.py [hunk 1]
```diff
@@ -106,7 +119,7 @@ def upgrade_entitlement_manifest_org(function_org, upgrade_entitlement_manifest,
     return function_org
 
 
-@pytest.fixture
+@pytest.fixture(scope='function')
```
Comment (Gauravtalreja1) [inline pytest_fixtures/component/taxonomy.py:122]: Default is function scope, so its the same thing

File: tests/foreman/endtoend/test_api_endtoend.py [hunk 2]
```diff
@@ -38,6 +38,7 @@
 from robottelo.config import settings
 from robottelo.config import user_nailgun_config
 from robottelo.constants.repos import CUSTOM_RPM_REPO
+from robottelo.utils import clone
```
Comment (Gauravtalreja1) [inline tests/foreman/endtoend/test_api_endtoend.py:41]: Isn't this useful somewhere?

File: tests/foreman/ui/test_contentview.py [hunk 3]
```diff
@@ -969,7 +969,9 @@ def test_positive_publish_with_rh_content(session):
 @pytest.mark.skip_if_not_set('fake_manifest')
 @pytest.mark.tier2
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_publish_composite_with_custom_content(session):
+def test_positive_publish_composite_with_custom_content(
+    session, function_entitlement_manifest_org, target_sat
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_contentview.py:973]: this test creates a org and uploads a manifest to it, not seeing any use of function_entitlement_manifest_org?

**Comment by shweta83** [comment]
It is no more valid so closing this PR.

**Pull request closed without merging**